docs/Makefile

```diff
@@ -3,22 +3,16 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = python -msphinx
 SPHINXPROJ    = FogLAMP
 SOURCEDIR     = .
 BUILDDIR      = _build
-CODEDIR       = ../src/python/foglamp
-HOSTADDRESS   = 0.0.0.0
-PORTNO        = 8000
 
 # Put it first so that "make" without argument is like "make help".
 help:
 	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-livehtml:
-	sphinx-autobuild -b html "$(SPHINXOPTS)" "$(BUILDDIR)/html" --host "$(HOSTADDRESS)" --port "$(PORTNO)" --watch "$(CODEDIR)"
-
-.PHONY: help livehtml Makefile
+.PHONY: help Makefile
```

Comment:
Again I would question if this was an intentional change.

Reply:
Default from Read the docs.